src/api/Data-API.coffee

```diff
@@ -14,8 +14,12 @@ class Data_API extends Swagger_GraphDB
     cache_Key = "article_#{ref}.json"
     @.open_Import_Service res, cache_Key, (import_Service)=>
       import_Service.graph_Find.find_Article ref, (article_Id)=>
-        data = { article_Id: article_Id}
-        @close_Import_Service_and_Send import_Service, res, data, cache_Key
+        if (article_Id?)
+          data = { article_Id: article_Id}
+          @close_Import_Service_and_Send import_Service, res, data, cache_Key
+        else
+          import_Service.graph.closeDb =>
+          res.send data?.json_pretty()
```

Comment:
Use the technique used here:

https://github.com/michaelhidalgo/TM_4_0_GraphDB/blob/master/src/api/Data-API.coffee#L39-L44